index.html

```diff
@@ -992,24 +993,37 @@ <h3>Overview</h3>
       <h3>Algorithm</h3>
 
       <p>This algorithm specifies how a new <a>active context</a> is updated
-        with a <a>local context</a>. The algorithm takes three input variables:
-        an <var>active context</var>, a <var>local context</var>, and an  <a>array</a>
-        <var>remote contexts</var> which is used to detect cyclical context inclusions.
-        If <var>remote contexts</var> is not passed, it is initialized to an empty
-        <a>array</a>.</p>
+        with a <a>local context</a>. The algorithm takes two required
+        <span class="changed">and two optional</span>
+        input variables:
```

Comment:
I would replace the colon by a period, and introduce the next sentence by "The requires inputs are".